msg/vehicle_status.msg

```diff
@@ -80,3 +80,10 @@ uint8 failure_detector_status			# Bitmask containing FailureDetector status [0,
 uint32 onboard_control_sensors_present
 uint32 onboard_control_sensors_enabled
 uint32 onboard_control_sensors_health
+
+# airspeed fault and airspeed use status
+bool aspd_check_failing		# true when airspeed consistency checks are failing
+bool aspd_fault_declared	# true when an airspeed fault has been declared
```

Comment:
Can we think about handling this similarly to position and velocity checks? https://github.com/PX4/Firmware/blob/master/msg/vehicle_status_flags.msg#L10

 - condition_airspeed_valid
 - similar probation logic?

Reply:
This PR is also publishing separate booleans so I don't understand the point you are making.